### DISCUSSION TITLE
up-switch unique selection

### DISCUSSION THREAD

**Discussion by ghost** [unpoly/unpoly#239] [Q&A] [answered]
Hi my friends 
I’m using up-switch in my form and it’s working fine 
![49247D34-1FB4-4450-AAFA-E02DE6875928](https://user-images.githubusercontent.com/86408288/127576250-fa2af8c2-373f-46d1-a6dc-38e7915ba2a8.jpeg)

my problem is when I select option in the first column the second select also appear in the other columns.

how can fixing that?!

I’m tried to add unique class to up-switch like this 

` <div up-switch=“.attribute-{{ field.id }}”> `

but the result is the same like this 

` <div up-switch=“.attribute-{{ field.id }}”> `

any suggestions to solve my problem

**Comment by adam12** [comment, marked as answer]
Can you re-create your issue in [Glitch](https://glitch.com/edit/#!/empty-unpoly-project)? It will be easier for us to understand.

**Reply by adam12** [reply]
@ilp94 I understand what you mean, but because Unpoly works on HTML markup, I'd need to see the markup to see why you're being affected. The easiest way to do that is to make a simple example of your issue.

I recreated one for you as an example, but I suggest you create one yourself next time. My guess is your selector in `up-switch` isn't unique enough, but without seeing the markup it can only be a guess.

Here's my working example:

Source: https://glitch.com/edit/#!/apple-mini-crown
Demo: https://apple-mini-crown.glitch.me/

**Reply by ghost** [reply]
thank you for example, but i'm already know that.
![r](https://user-images.githubusercontent.com/86408288/127667489-23cb79d3-23cf-493f-b84c-bae20d4b822e.png)
i'm using django for loop to makes multi rows.
 i'm trying to add unique id to html class but it didn't render inside up-switch.
![u](https://user-images.githubusercontent.com/86408288/127669330-9ce10a87-fe51-43a8-9d17-1781888d038d.png)

**Reply by adam12** [reply]
Since I see `{{ field.id }}` in the markup you posted, it seems like this is related to Django and not Unpoly.

**Reply by ghost** [reply]
yes, the problem was in django widget tweak library, i used it to render my form's fields
thank you dear
